src/server/templates/settings.html

```diff
@@ -1556,6 +1556,11 @@
 			return false;
 		});
 
+		$('button#kill_server').click(function (event) {
+			socket.emit('kill_server');
+			return false;
+		});
+
```

Comment:
Do the changes that allow this also make the 'Restart' option functional that was explored in the `restart-self` branch? If not directly, maybe they provide a path to making it happen? 74384f0

Reply:
Not sure.

Reply:
Outside the scope of the PR anyway. Proceed and we should revisit later.